setup.py

```diff
@@ -18,7 +17,7 @@
     ]
 }
 DESCRIPTION = (
-    'renders pyecharts as image' +
+        'renders pyecharts output file as image' +
```

Comment:
sorry but this change should go to pyecharts-snapshot.yml, 

moban will generate setup.py again :)